pkg/controller/plan/migration.go

```diff
@@ -1638,6 +1647,19 @@ func (r *Migration) updatePopulatorCopyProgress(vm *plan.VMStatus, step *plan.St
 	return
 }
 
+func (r *Migration) isPopulatorFailed(populatorPods []core.Pod, givenPvcId string) error {
```

Comment:
that's a bit awkward to have a function called `isPopulatorFailed` that returns an error - usually an error indicates that an error was raised when executing the function.. how about returning a boolean, or a boolean with an error if we need the error?